docs/api.md

```diff
@@ -1273,8 +1273,8 @@ or updates existing device details for this user/session
 If no device `id` is specified,
 both `name` and `type` must be provided.
 If a device `id` is specified,
-at least one of `name`, `type`, `pushCallback` and `pushPublicKey`
-must be present.
+at least one of `name`, `type`, `pushCallback` or the couple `pushPublicKey` and `pushAuthKey`
+must be present. Be careful, `pushPublicKey` and `pushAuthKey` can change if `pushCallback` changes
```

Comment:
The only time would be if you were doing a non-data bearing push update, in which case, you'd only need the pushCallback. Otherwise, you'll want to keep all three values since you'll need those to send data for a given user.